src/auth/components/game-img/game-img.vue

```diff
@@ -0,0 +1,108 @@
+import { ThemeStore } from '../../../_common/theme/theme.store';
+<template>
+	<div id="cover-img-credits">
+		<div class="flow-col">
+			<a class="link-unstyled -title" :href="gameUrl">{{ gameTitle }}</a>
```

Comment:
always use router-link, you would just change `href` to `to`: `<router-link class="-title link-unstyled" :to="gameUrl">...`